docs/run-instance/data.md

```diff
@@ -29,6 +29,21 @@ Data relating to the avoid borders features is derived from administrative bound
 Configuration parameters: [
 `ors.engine.profiles.<PROFILE-NAME>.ext_storages.Borders`](configuration/engine/profiles/build.md#borders)
 
+To ensure `avoid_borders` functions correctly, the following are required:
+
+- **Field names and encoding:**  
+  `borders.geojson` must include a `name` property.  
+  In `ids.csv`, the `name` column must use consistent Latin/ASCII names.  
```

Comment:
I don't think this is actually required. For example, this is how an example entry for one of the countries looks like across [the files in production](https://gitlab.heigit.org/giscience/openrouteservice-infrastructure/ors-docker-server-deploy-v2/-/tree/main/instances/extras/borders?ref_type=heads).

**ids_iso.csv**
```
"country_id","name","name:en","iso_cca2","iso_cca3"
...
"198","ประเทศไทย","Thailand","TH","THA"
...
```

**boundaries.geojson**
```
...
{ "type": "Feature", "properties": { "name": "ประเทศไทย", "hierarchy": 249 }, "geometry": { "type": "MultiPolygon", "coordinates": [ [ [ [ 98.99999, 16.875182 ], [ 99.99999, 16.875182 ], [ 99.99999, 14.875182 ], [ 98.99999, 14.875182 ], [ 98.99999, 15.875182 ], [ 98.99999, 16.875182 ] ] ] ] } },
...

Reply:
Rewritten to accommodate for these details.